doc/theoretical_description_regression.rst

```diff
@@ -178,6 +178,41 @@ methods and emphasizes their main differences.
    :width: 800
 
 
+7. The jackknife+-after-bootstrap method
+========================================
+
+In order to reduce the computational time, and get more robust predictions, 
+one can adopt a bootstrap approach instead of a leave-one-out approach, called 
+the jackknife+-after-bootstrap method, offered by Kim and al. [2].
+
+By analogy with the CV+ method, estimating the prediction intervals with 
+jackknife+-after-bootstrap is performed in four main steps:
+
+- We resample the training set with replacement (boostrap) :math:`K` times,
+  and thus we get the (non disjoint) bootstraps :math:`B_{1},..., B_{K}` of equal size.
+
+
+- :math:`K` regressions functions :math:`\hat{\mu}_{B_{k}}` are then fitted on 
+  the bootstraps :math:`(B_{k})`, and the predictions on the complementary sets 
+  :math:`(B_k^c)` are computed.
+
+
+- These predictions are aggregated according to a given aggregation function 
+  :math:`agg`, typically :math:`mean` or :math:`median`, and the residuals 
+  :math:`|Y_j - agg(\hat{\mu}(B_{K(j)}(X_j)))|` are computed for each :math:`X_j`
```

Comment:
Could you please write "agg" as something like {\rm agg} in an equation so it's not written in italics ?

Reply:
@tmorzade, seems ok now ?

Reply:
Done